src/v2/BaseLoginRouter.js

```diff
@@ -224,7 +226,10 @@ export default Router.extend({
     // state token (which will be set into AppState)
     if (this.settings.get('oieEnabled')) {
       try {
-        const idxResp = await startLoginFlow(this.settings);
+        let idxResp = await startLoginFlow(this.settings);
+        if (this.appState.get('isFirstRender')) {
```

Comment:
Can `identifier first` + `reset password` be handled here? iirc, the action will only come back in the second call (the response of `identify.proceed()`)

Reply:
I believe the identify first flows may break with this implementation. At this point, I think that is OK. identifier-first flows can be more easily supported when backend support